### PR TITLE
feat(rnafusion) Add rnafusion as an ordertype

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
Adds functionality to be able to submit rnafusion order via the API

### Added

- Support for placing an order via `api/v1/submit_order/rnafusion`
- Support for uploading excel orderforms with rnafusion as data analysis

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [x] Deploy the branch to `cg-vm1`

### How to test

- [x] Post a request to `https://cg-statusdb-stage.scilifelab.se/api/v1/submit_order/rnafusion` with a valid rnafusion order
- [x] Post a request to `https://cg-statusdb-stage.scilifelab.se/api/v1/orderform` with a valid 1508 orderform with rnafusion as the data analysis.

### Expected test outcome

- [x] Check samples are correctly added to statusdb and lims.
- [x] Check that the excel orderform can be parsed and correctly returned.

## Review

- [x] Tests executed by VJ
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/meta/orders/conftest.py [hunk 1]
```diff
@@ -21,20 +23,22 @@
     mip_order_to_submit,
     mip_rna_order_to_submit,
     rml_order_to_submit,
+    fixture_rnafusion_order_to_submit,
     sarscov2_order_to_submit,
 )
 
 
 @pytest.fixture(scope="session", name="all_orders_to_submit")
 def fixture_all_orders_to_submit(
```
Comment (ChrOertlin) [inline tests/meta/orders/conftest.py:32]: docstring

File: tests/meta/orders/conftest.py [hunk 2]
```diff
@@ -21,20 +23,22 @@
...
+    balsamic_order_to_submit: dict,
+    fastq_order_to_submit: dict,
+    metagenome_order_to_submit: dict,
+    microbial_order_to_submit: dict,
+    mip_order_to_submit: dict,
+    mip_rna_order_to_submit: dict,
+    rml_order_to_submit: dict,
+    rnafusion_order_to_submit: dict,
+    sarscov2_order_to_submit: dict,
 ):
```
Comment (ChrOertlin) [inline tests/meta/orders/conftest.py:42]: return typehint

**Review by ChrOertlin — APPROVED** [review]
looks good, very minor comments

**Comment by Vince-janv** [comment]
### rnafusion order accepted:
Screenshot 2023-06-02 at 09 09 50

**Pull request merged**